<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomizeKitDialog</class>
 <widget class="QDialog" name="CustomizeKitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customize Kit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTreeWidget" name="pitchList">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>56</number>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>20</number>
       </attribute>
       <column>
        <property name="text">
         <string extracomment="Number">No.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Note</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Shortcut</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>name</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="name">
         <property name="placeholderText">
          <string>Enter a name to enable pitch editing</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Notehead group:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>noteHead</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="noteHead"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="customGbox">
         <property name="title">
          <string>Edit noteheads</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="quarterLbl">
            <property name="text">
             <string>Quarter note</string>
            </property>
            <property name="buddy">
             <cstring>quarterCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="quarterCmb"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="halfLbl">
            <property name="text">
             <string>Half note</string>
            </property>
            <property name="buddy">
             <cstring>halfCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="halfCmb"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="wholeLbl">
            <property name="text">
             <string>Whole note</string>
            </property>
            <property name="buddy">
             <cstring>wholeCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="wholeCmb"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="doubleWholeLbl">
            <property name="text">
             <string>Double whole note</string>
            </property>
            <property name="buddy">
             <cstring>doubleWholeCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="doubleWholeCmb"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="mu::palette::PaletteWidget" name="drumNote" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Default voice:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>voice</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="voice">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Staff line:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>staffLine</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="staffLine">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>-12</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Shortcut:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>shortcut</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="shortcut">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string notr="true">A</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">B</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">C</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">D</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">E</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">F</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">G</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">--</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Stem direction:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>stemDirection</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="stemDirection">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Auto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Up</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Down</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load…</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save as…</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mu::palette::PaletteWidget</class>
   <extends>QWidget</extends>
   <header>palette/view/widgets/palettewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pitchList</tabstop>
  <tabstop>name</tabstop>
  <tabstop>noteHead</tabstop>
  <tabstop>customGbox</tabstop>
  <tabstop>quarterCmb</tabstop>
  <tabstop>halfCmb</tabstop>
  <tabstop>wholeCmb</tabstop>
  <tabstop>doubleWholeCmb</tabstop>
  <tabstop>voice</tabstop>
  <tabstop>staffLine</tabstop>
  <tabstop>shortcut</tabstop>
  <tabstop>stemDirection</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomizeKitDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomizeKitDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
